docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingSample.cs

```diff
@@ -10,71 +13,88 @@ public static void Example()
             var mlContext = new MLContext();
 
             // Get a small dataset as an IEnumerable and convert it to an IDataView.
-            var data = SamplesUtils.DatasetUtils.GetInfertData();
-            var trainData = mlContext.Data.LoadFromEnumerable(data);
+            var rawData = GetData();
+
+            // Printing the input data. 
+            Console.WriteLine("Age\t Salary");
+            foreach (var row in rawData)
+                Console.WriteLine($"{row.Age}\t {row.Salary}");
+            // Expected output:
+            // Age      Salary
+            // 26       40000
+            // 35       80000
+            // 34       10000
+            // 28       100000
 
-            // Preview of the data.
-            //  Age     RowNum  Education   ...
-            //  26       0       0-5yrs     ...
-            //  42       1       0-5yrs     ...
-            //  39       2       12+yrs     ...
-            //  34       3       0-5yrs     ...
-            //  35       4       6-11yrs    ...
+            var data = mlContext.Data.LoadFromEnumerable(rawData);
 
             // We define the custom mapping between input and output rows that will be applied by the transformation.
-            Action<SamplesUtils.DatasetUtils.SampleInfertData, OutputRow> mapping =
+            Action<InputData, CustomMappingOutput > mapping =
                 (input, output) => output.IsUnderThirty = input.Age < 30;
 
-            // Custom transformations can be used to transform data directly, or as part of a pipeline. Below we transform data directly.
-            var estimator = mlContext.Transforms.CustomMapping(mapping, null);
-            var transformedData = estimator.Fit(trainData).Transform(trainData);
+            // Custom transformations can be used to transform data directly, or as part of a pipeline of estimators.
+            // Note: If contractName is null in the CustomMapping estimator, any pipeline of estimators containing it, cannot be saved and loaded back. 
+            // See other sample on how to load and save the CustomMapping estimator.
+            var estimator = mlContext.Transforms.CustomMapping(mapping, contractName: null);
```

Comment:
>estimator [](start = 16, length = 9)

we call these 'pipeline' in the rest of sample, eventhough the only have a single estimator